src/themes/teams/components/Input/inputVariables.ts

```diff
@@ -1,21 +1,31 @@
 import { pxToRem } from '../../../../lib'
+export interface IInputVariables {
+  borderRadius: string
+  borderBottom: string
+  backgroundColor: string
+  fontColor: string
+  fontSize: string
+  iconPosition: string
+  iconRight: string
+  inputPadding: string
+  inputFocusBorderColor: string
+  inputFocusBorderRadius: string
+}
 
-export default (siteVars: any) => {
-  const vars: any = {}
-
-  vars.borderRadius = `${pxToRem(3)}`
-  vars.borderBottom = `${pxToRem(2)} solid transparent`
-  vars.backgroundColor = siteVars.gray10
+const [px2asRem, px3asRem, px6asRem, px12asRem, px24asRem] = [2, 3, 6, 12, 24].map(v => pxToRem(v))
```

Comment:
maybe the following naming pattern will result in more readable strings at the place of usage:

```js
px2asRem -> _2px_asRem

///
inputFocusBorderRadius: `${px3asRem} ${px3asRem} ${px2asRem} ${px2asRem}`,

// vs
inputFocusBorderRadius: `${_3px_asRem} ${_3px_asRem} ${_2px_asRem} ${_2px_asRem}`,
```